clients/vault/tests/vault_integration_tests.rs

```diff
@@ -608,6 +608,18 @@ async fn test_issue_cancel_succeeds() {
 #[serial]
 async fn test_issue_execution_succeeds_from_archive() {
 	let is_public_network = true;
+	do_test_issue_execution_succeeds_from_archive(is_public_network).await;
+}
+
+#[tokio::test(flavor = "multi_thread")]
+#[ntest::timeout(1_200_000)] // timeout at 20 minutes
+#[serial]
+async fn test_issue_execution_succeeds_from_archive_testnet() {
+	let is_public_network = false;
+	do_test_issue_execution_succeeds_from_archive(is_public_network).await;
+}
+
+async fn do_test_issue_execution_succeeds_from_archive(is_public_network: bool){
```

Comment:
Nit: Adhering to the naming schema of the other functions let's call them

```suggestion
#[serial]
async fn test_issue_execution_succeeds_from_archive_on_mainnet() {
	let is_public_network = true;
	test_issue_execution_succeeds_from_archive_on_network(is_public_network).await;
}

#[tokio::test(flavor = "multi_thread")]
#[ntest::timeout(1_200_000)] // timeout at 20 minutes
#[serial]
async fn test_issue_execution_succeeds_from_archive_on_testnet() {
	let is_public_network = false;
	test_issue_execution_succeeds_from_archive_on_network(is_public_network).await;
}

async fn test_issue_execution_succeeds_from_archive_on_network(is_public_network: bool){
```